app/src/main/java/net/tensory/rxjavatalk/data/DebtFeed.java

```diff
@@ -33,15 +34,15 @@ public DebtFeed() {
                         ThreadLocalRandom.current().nextDouble(MAX_DEBT_GOLD));
     }
 
-    public BehaviorSubject<Double> getLannisterDebt() {
-        return lannisterDebt;
+    public Observable<Double> getLannisterDebt() {
```

Comment:
Just improving the data encapsulation, this makes sure that we don't leak the BehaviorSubject outside of this class, plus we get back pressure, yay!

Reply:
Oops, thanks!